src/ray/pubsub/publisher.h

```diff
@@ -73,17 +74,17 @@ class EntityState {
   // the message in buffer.
   std::queue<std::weak_ptr<rpc::PubMessage>> pending_messages_;
   // Size of each inflight message.
-  std::queue<int64_t> message_sizes_;
+  std::queue<size_t> message_sizes_;
   // Protobuf messages fail to serialize if 2GB or larger. Cap published
   // message batches to this size to ensure that we can publish each message
   // batch. Individual messages larger than this limit will also be dropped.
   // TODO(swang): Pubsub clients should also ensure that they don't try to
   // publish messages larger than this.
-  const int64_t max_message_size_bytes_;
+  const size_t max_message_size_bytes_;
```

Comment:
Can  you change `RAY_CONFIG(int64_t, max_grpc_message_size, 512 * 1024 * 1024)` to `size_t` as well?

Reply:
done